cmd/ulimit_unix.go

```diff
@@ -41,11 +42,5 @@ func setUlimit(ulimit uint64) error {
 		rLimit.Cur = ulimit
 	}
 
-	return syscall.Setrlimit(syscall.RLIMIT_NOFILE, &rLimit)
-}
-
-func GetUlimit() (syscall.Rlimit, error) {
-	var rLimit syscall.Rlimit
-	err := syscall.Getrlimit(syscall.RLIMIT_NOFILE, &rLimit)
-	return rLimit, err
+	return syscall.Setrlimit(syscall.RLIMIT_NOFILE, &syscall.Rlimit{Cur: rLimit.Cur, Max: rLimit.Max})
```

Comment:
maybe this should be moved to the SDK. Also removing the need to seperate ulimit files, and where we call it in root just do limit.Set etc' and in windows it will do nothing etc'.

Reply:
Makes sense, I was able to make the test fail so looking into it